### DISCUSSION TITLE
Problem with create row automation with Budibase db

### DISCUSSION THREAD

**Discussion by mattdeloach** [Budibase/budibase#3710] [Help]
Hello, I'm very new to budibase so apologies if this is blatantly obvious but I'm having trouble creating a new row in a budibase db with an automation. I've tried various triggers to initiate the automation but I can't get past the initial test step without incurring a "TypeError: Pouch is not a constructor" error in the response

![image](https://user-images.githubusercontent.com/1061546/145096926-943ff5a1-f3b0-4a0f-add3-e084eff5a9ac.png)
![image](https://user-images.githubusercontent.com/1061546/145097008-5ed56064-3e84-4cbe-950b-44023c3cc060.png)

Any information or documentation would be greatly appreciated. I haven't been able to find a detailed walkthrough or information on a Pouch error. Thank you!

**Comment by JMLX42** [comment]
Same problem here **but** the row actually **is** added despite the error. So this is very misleading.

**Comment by LaskiBear** [comment]
Im getting same in query rows and any other activity that relates to CouchDB. 
Havent noticed that create row even creates row despite error. But query wont return any rows.

**Comment by mjashanks** [comment]
Hi @LaskiBear @mattdeloach  @JMLX42 

Apologies for this - we have fixed this internally and will deploy to cloud and self-host today

**Reply by LaskiBear** [reply]
Should this be fixed for version 1.0.9 ? Just got it to self hosted environment but still getting same error

**Reply by mjashanks** [reply]
Not yet - I'll post in here when we've released it.

**Reply by JMLX42** [reply]
> Apologies for this - we have fixed this internally and will deploy to cloud and self-host today

@mjashanks outstanding reactivity! Ping me whenever I can test it locally and I'll report back here ASAP to confirm it's working.

**Reply by mjashanks** [reply]
Fix pushed to self-host and cloud.

**Reply by LaskiBear** [reply]
Still getting same error im on version 1.0.11 and no updates are available ?

**Reply by mjashanks** [reply]
Hey @LaskiBear - could you recreate your automation? This was an issue to do with bindings - I think your existing bindings will still be broken, even though we've fixed the cause.

Apologies, I didn't realise that this needed to be done to fix it.

**Reply by LaskiBear** [reply]
Yeah i tested that all ready created new automation and same error

**Reply by JMLX42** [reply]
I recreated the automation and I still have the same error:

```json
{
  "success": false,
  "response": "TypeError: Pouch is not a constructor"
}
```

```
docker images
REPOSITORY                                              TAG                                     IMAGE ID       CREATED         SIZE
budibase.docker.scarf.sh/budibase/apps                  latest                                  85acf3df0ed1   21 hours ago    1.8GB
budibase.docker.scarf.sh/budibase/worker                latest                                  1b080e63038b   21 hours ago    771MB
```

**Reply by mjashanks** [reply]
ok, thank you for trying this... will investigate.

**Reply by mattdeloach** [reply]
I just deleted and recreated the automation with the same Pouch error as well on the cloud

**Reply by mjashanks** [reply]
Sorry all, we'll have a look at this again on Monday.

**Reply by mattdeloach** [reply]
Thank you @mjashanks! Any updates on this by chance?

**Reply by LaskiBear** [reply]
This is realy starting to be stopper for develop, any news on this matter ?

**Reply by mjashanks** [reply]
Hi All, We've fixed this (again) and will get it released over the next few days.

**Reply by mjashanks** [reply]
@LaskiBear  @mattdeloach  @JMLX42 - this has been fixed and released to self-host and cloud.

I have verified that it is fixed in cloud.

**Reply by mattdeloach** [reply]
@mjashanks Thank you very much! It's working for me. I appreciate you getting that fix deployed!
